src/components/NavigationBar/index.js

```diff
@@ -6,6 +6,8 @@ import NavigationTitle from './Title';
 import NavigationSide from './Side';
 import Inputs from '../Inputs/Inputs';
 
+import * as colors from '../../style/vars/colors';
```

Comment:
Naming convention isn't consistent here. You use `Colors` elsewhere.

Reply:
good catch. made the change